CONTRIBUTING.md

```diff
@@ -1,3 +1,15 @@
 # Contributing
 
 We love pull requests from everyone. By participating in this project, you agree to abide by the Web Matters [code of conduct](http://www.web-matters.co.uk/code-of-conduct.html).
+
+## Process
+
+The site is run by GitHub from `master` so do not directly commit to `master`.
+
+We operate a two phase commit policy so all code should be reviewed before being merged.
```

Comment:
`We operate a two-phase commit policy, so all code should be reviewed before being merged.` - suggested grammar improvement.